postgres/Dockerfile

```diff
@@ -20,6 +20,8 @@ RUN cd /usr/src/pg-spgist_hamming/bktree \
 
 FROM postgres:15-alpine
 
+COPY ./postgresql.conf /etc/postgresql/postgresql.conf
+
```

Comment:
I left it out so users can configure Postgres based on their machine specs. They can add their config to the container later.